src/components/CardList/Card.jsx

```diff
@@ -0,0 +1,89 @@
+import React from 'react';
+import styled from 'styled-components';
+import SelectOpt from './SelectOpt';
+
+const Card = ({ product, selectOpt }) => {
+  const { title, images, price, productStatus } = product;
+
+  const returnStatusBtn = () => {
+    if (productStatus === 'selling') {
+      return null;
+    }
+    let obj = {
+      bgColor: productStatus === 'soldout' ? '#333' : '#486344',
+      text: productStatus === 'soldout' ? '판매 완료' : '예약중',
+    };
+
+    return <ProductStatus bgColor={obj.bgColor}>{obj.text}</ProductStatus>;
+  };
+
+  return (
+    <CardWrapper>
+      <ImgWrapper>
+        <ProductImg src={images[0]} alt="product" />
+        {productStatus === 'soldout' && <Dimd />}
+        {returnStatusBtn()}
```

Comment:
```suggestion
       {isProductSoldout && <Dimd />}
        {!isProductSelling && (
          <ProductStatus bgColor={productStatusOption.bgColor}>
            {productStatusOption.text}
          </ProductStatus>
        )}
```

위의 리뷰와 함께 묶어서 말씀드리면 함수의 return 값을 통해 컴포넌트를 배출하는 방법보다
위와 같은 방식으로 작성하는 것이 UI와 로직을 분리해서 구현하는 방법이라고 생각 되네요!

Reply:
안그래도 이부분 고민이 많았는데 리뷰 감사합니다!

Reply:
한가지 더 궁금한점이 
마이페이지 > 판매 목록 탭에서 상태변경하는 경우가 있어서
이부분을 프론트에서 처리하려면 <List /> 에서 fetch가 아닌 
상위에서 fetch 후 데이터를 props로 받아서 처리해야 하는걸까요??

Reply:
답변 감사합니다!